### ISSUE TITLE
Create shortcut for Save selection

### ISSUE THREAD

**Issue by greengeek** [gildas-lormeau/SingleFile#1215]
**Is your feature request related to a problem? Please describe.**
Is there a setting to change the default action of the SingleFile browser icon to perform one of the alternate functions, such as Save selection?

**Describe the solution you'd like**
Setting to change what happens when one clicks the SingleFile browser icon, such as Save Page, Save Links, Save Selection.  It would also be nice to be able to set a keyboard shortcut to the Save selection function.

**Comment by gildas-lormeau** [comment]
I agree that's a desirable feature. For the record, it's similar to https://github.com/gildas-lormeau/SingleFile/issues/700. Both ideas could be combined though.
